web/styles/settings.css

```diff
@@ -1308,7 +1308,9 @@ label.preferences-radio-choice-label {
         .tab-container {
             box-sizing: border-box;
             height: $settings_header_height;
-            padding: 6px;
+            display: flex;
+            align-items: center;
+            justify-content: center;
```

Comment:
The padding isn't necessary anymore because the contents are centered with flex. Previously they were un-centeredly placed with padding.